tools/emdwp.py

```diff
@@ -12,7 +12,7 @@
 
 __scriptdir__ = os.path.dirname(os.path.abspath(__file__))
 __rootdir__ = os.path.dirname(__scriptdir__)
-sys.path.append(__rootdir__)
+sys.path.insert(1, __rootdir__)
```

Comment:
Why not insert at 0 so it actually the first element?

Reply:
I did a print of the sys.path earlier
` ['/home/yapt/myproject/components', '/usr/lib/python38.zip', '/usr/lib/python3.8', '/usr/lib/python3.8/lib-dynload', '/home/yapt/myproject/bin/Debug/AssemblyRef/python/lib/python3.8/site-packages']`

The element 0 stores the path of the current directory the current script resides in.
Hence it would be idea for the search path to start from where the script resides in, follow by emscripten's root directory.

Reply:
Why?   Shouldn't we always prefer the emscripten root? 

Also, sys.path[0] might be something different because somebody else might have done an insert(0) in their python startup script, right?

I think using 0 is more clear what we are trying to achieve.

Reply:
Noted will change them into element 0.